util/customTransformer-v1.js

```diff
@@ -30,9 +34,23 @@ async function transform(isolatevm, events) {
       reject(error.message);
     }
   });
-  return executionPromise.catch(e => {
-    throw new Error(e);
-  });
+
+  let result;
+  try {
+    const timeoutPromise = new Promise(resolve => {
+      const wait = setTimeout(() => {
+        clearTimeout(wait);
+        resolve("Timedout");
```

Comment:
it is possible to reject directly from here right? Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/race
if we do this, we may not need to wrap into try-catch block

Reply:
made the changes